packages/workbox-build/test/node/get-file-manifest-entries.js

```diff
@@ -217,9 +217,9 @@ describe('Test getFileManifestEntries', function() {
       globDirectory: path.join(__dirname, '..', '..', '..',
         'workbox-cli', 'test', 'static', 'example-project-1'),
       templatedUrls: {
-        'template/url1': ['page-1.html', 'index.html'],
-        'template/url2': ['page-2.html', 'index.html'],
-        'template/url3': '<html><head></head><body><p>Just in case</p></body></html>',
+        '/template/url1': ['page-1.html', 'index.html'],
+        '/template/url2': ['page-2.html', 'index.html'],
+        '/template/url3': '<html><head></head><body><p>Just in case</p></body></html>',
```

Comment:
@gauntface just so that I am clear - if we make the url3 (last one) relative wouldn't it give a problem when the html content it is referring to is updated